src/patternfly/components/ClipboardCopy/clipboard-copy.scss

```diff
@@ -41,15 +45,18 @@
   --pf-c-clipboard-copy__expandable-content--OutlineOffset: calc(-1 * var(--pf-global--spacer--xs));
 
   // Toggle button expanded
-  &.pf-m-expanded .pf-c-clipboard-copy__group-toggle {
-    &::before {
-      --pf-c-clipboard-copy__group-toggle--BorderBottomColor: var(--pf-c-clipboard-copy__group-toggle--m-expanded--BorderBottomColor);
-
-      border-bottom-width: var(--pf-c-clipboard-copy__group-toggle--m-expanded--BorderBottomWidth);
-    }
-
-    .pf-c-clipboard-copy__group-toggle-icon {
-      transform: rotate(90deg);
+  &.pf-m-expanded {
+    .pf-c-clipboard-copy__group-toggle {
+      &::before {
+        --pf-c-clipboard-copy__group-toggle--BorderBottomColor: var(--pf-c-clipboard-copy__group-toggle--m-expanded--BorderBottomColor);
+
+        border-bottom-width: var(--pf-c-clipboard-copy__group-toggle--m-expanded--BorderBottomWidth);
+      }
+
+      &:focus .pf-c-clipboard-copy__group-toggle-icon {
+        transition: var(--pf-c-clipboard-copy__group-toggle-icon--m-expanded--Transition);
```

Comment:
you'll want to move the `transition` here, too, so it applies to `.pf-c-clipboard-copy__group-toggle-icon` on it's own, not just in the context of `pf-m-expanded`. Also can we update the var name so it matches the selector here, too? And create a follow up issue with a breaking change label to address the existing vars we have where the var name doesn't match the selectors?